scvi/hub/model_card_template.py

```diff
@@ -0,0 +1,45 @@
+template = """\
+---
+{card_data}
+---
+
+# Description
+
+{description}
+
+# Model properties
+
+Many model properties are in the model tags. Some more are listed below.
+
+**cell_count**: {cell_count}
+
+**gene_count**: {gene_count}
+
+**model_init_params**:
+```json
+{model_init_params}
+```
+
+**model_setup_anndata_args**:
+```json
+{model_setup_anndata_args}
+```
+
+**model_parent_module**: {model_parent_module}
+
+**data_is_latent**: {data_is_latent}
+
+# Large data
+
+This is an optional link to where the training data is stored if it is too large
+to host on the huggingface Model hub.
+
+<!-- This field is required for models that haven't been minified by converting to latent
+mode. See the scvi-tools documentation for more details. -->
+
+Large data url: {large_data_url}
+
```

Comment:
make a note here that if you want to update the url, to update the json file as well

Reply:
emphasize in tutorial that json should be up to date

Reply:
Took a note to mention this in the tutorials when I add them